src/Formatter/NMEAFormatter.hpp

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2022 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+#pragma once
+
+#include "NMEA/Info.hpp"
+
+/**
+ * GPRMC,<1>,<2>,<3>,<4>,<5>,<6>,<7>,<8>,<9>,<10>,<11>
+ * 
+ * <1>  UTC Time of position fix, hhmmss format
+ * <2>  Status, A = Valid position, V = NAV receiver warning
+ * <3>  Latitude,ddmm.mmmm format (leading zeros will be transmitted)
+ * <4>  Latitude hemisphere, N or S
+ * <5>  Longitude,dddmm.mmmm format (leading zeros will be transmitted)
+ * <6>  Longitude hemisphere, E or W
+ * <7>  Speed over ground, 0.0 to 999.9 knots
+ * <8>  Course over ground 000.0 to 359.9 degrees, true
+ *      (leading zeros will be transmitted)
+ * <9>  UTC date of position fix, ddmmyy format
+ * <10> Magnetic variation, 000.0 to 180.0 degrees
+ *      (leading zeros will be transmitted)
+ * <11> Magnetic variation direction, E or W
+ *      (westerly variation adds to true course)
+ */
+bool FormatGPRMC(TCHAR *buffer, size_t size, const NMEAInfo &info);
```

Comment:
Code-style issue.

Reply:
Ok, will check and update (and also fix the build errors for the tests).